### ISSUE TITLE
`isoOffset` fails to calculate GMT+5:45 Asia/Kathmandu timezone, returns buggy output

### ISSUE THREAD

**Issue by sudiptochoudhury** [spencermountain/spacetime#226]
```
const date = spacetime(null, 'Asia/Kathmandu');
console.log(date.format('offset'));
```

Shows:

`"+5.75:00"`

**Comment by spencermountain** [comment]
Ah, good find. Will fix this week. Thanks Sudipto.

**Comment by spencermountain** [comment]
released this as `6.6.4` - thank you!
